src/main/features/claim/views/claimant-individual-details.njk

```diff
@@ -27,7 +27,8 @@
               name: 'address',
               postcodeLookup: true
             },
```

Comment:
Please make use of this pull request. https://github.com/hmcts/cmc-citizen-frontend/pull/2458
I have added label to each input and you can use span with invisible label and remove the aria-label.

Reply:
Yes i have reverted my changes and updated the PR-2458 in the ticket.